doc/SAI_1.7.1 Release notes.md

```diff
@@ -0,0 +1,149 @@
+
+The PR related to this feature is available at [PR#1158](https://github.com/opencomputeproject/SAI/pull/1158)
+
+### Add neighbor and tunnel term ip addr family for CRM 
+Generic resource monitoring, done by API sai_object_type_get_availability, introduced the ability to query avaialble resources per object type, and provide additional attributes as query params. This introduces 2 such params - neighbor entry IP address family, to be able to query avaialble IPv4 and IPv6 neigbor entries, and tunnel termination table entry IP address family, to be able to query available IPv4 and IPv6 tunnel termination table entries. Introduced attributes are read only, marked as resourcetype, and used only for this query by adding attribute SAI_NEIGHBOR_ENTRY_ATTR_IP_ADDR_FAMILY in enum sai_neighbor_entry_attr_t for saineighbor.h and attribute SAI_TUNNEL_TERM_TABLE_ENTRY_ATTR_IP_ADDR_FAMILY in enum sai_tunnel_term_table_entry_attr_t for saitunnel.h
+
+The PR related to this feature is available at [PR#1151](https://github.com/opencomputeproject/SAI/pull/1151)
+
+### UDF Field Qualifier in ACL 	
+This feature brings in change to be able to specify field extracted by UDF group as a qualifier in ACL Table for typedef enum _sai_acl_table_attr_t 
+
+The PR related to this feature is available at [PR#1122](https://github.com/opencomputeproject/SAI/pull/1122)
+
+### Packet loopback action for tunnels 
+Added loopback action as switch attribute: This will apply to all the tunnels configured and as a tunnel specific action. Added SAI_OUT_DROP_REASON_TUNNEL_LOOPBACK_PACKET_DROP under typedef enum _sai_out_drop_reason_t for saidebugcounter.h and  SAI_SWITCH_ATTR_TUNNEL_LOOPBACK_PACKET_ACTION, under typedef enum _sai_switch_attr_t for saiswitch.h and SAI_TUNNEL_ATTR_LOOPBACK_PACKET_ACTION under typedef enum _sai_tunnel_attr_t for saitunnel.h 
+
+The PR related to this feature is available at [PR#1152](https://github.com/opencomputeproject/SAI/pull/1152)
+
+### Allow for multiple interface types  
+This allows for provisioning and advertising a list of multiple IEEE ethernet protocols on a port, by adding SAI_PORT_ATTR_ADVERTISED_INTERFACE_TYPE in enum sai_port_attr_t for saiport.h
+
+The PR related to this feature is available at [PR#1135](https://github.com/opencomputeproject/SAI/pull/1135)
+
+### New Interface type based on transceivers variants  
+New interface types as been added for saiport.h under enum sai_port_interface_type_t 
+
+The PR related to this feature is available at [PR#1139](https://github.com/opencomputeproject/SAI/pull/1139)
+
+### Add next hop group member counter	    
+This features adds the counter to attach a SAI counter to next hop group member, to count traffic routed to it by adding attribute SAI_NEXT_HOP_GROUP_MEMBER_ATTR_COUNTER_ID in enum sai_next_hop_group_member_attr_t for sainexthopgroup.h

```

Comment:
probably a typo, adds the ability to attach

Reply:
addressed